doc/man/bitcoind.1

```diff
@@ -456,6 +456,9 @@ Set maximum size of high\-priority/low\-fee transactions in bytes
 .IP
 Set lowest fee rate (in BTC/kB) for transactions to be included in block
 creation. (default: 0.00001)
+\fB\-bip148\fR
```

Comment:
I'm not suggesting different functionality. Sorry if I wasn't clear. What I am suggesting is that the best way to protect users from a reorg is by switching to a segwit signaling chain (This PR).

It's best to be in a situation where users only need to switch a simple flag rather than download a brand new (maybe rushed) release.

But some are fighting the idea of bip148 on bitcoind. So what I am suggesting is a simple name change to put things into perspective to avoid that argument.

The perspective being: Users need a flag to prevent reorg so enforcing bip148 isn't about agreeing with bip148 but instead to protect their node from reorg.